modules/nextflow/src/main/groovy/nextflow/Session.groovy

```diff
@@ -468,13 +468,27 @@ class Session implements ISession {
         CH.broadcast()
 
         if( preview ) {
+            dumpContainerImages()
```

Comment:
It would be better to enable this using a dedicated option that implicitly enable *preview*